### PR TITLE
Update Kromer-plot tools

### PR DESCRIPTION
**Pull request by unoebauer**
## Overview

The original Kromer plotting script was not very elegant and utterly undocumented. Also, it doesn't run out of the box with the current master (fe26323be95b44a948c98e5b026a714628208b84). This PR aims at a major overhaul of the plotting script.

## Roadmap

- [x] make fully OO
- [x] add documentation
- [x] compatibility with current Tardis master
- [x] PEP8 compliance

### CODE REVIEW

**Comment by unoebauer** [comment]
@wkerzendorf 
Any comments on this - otherwise, I'd merge this one since @chvogl wants to adapt it to the continuum runs

File: tardis_kromer_plot.py [hunk 1]
```diff
@@ -10,212 +13,584 @@
...
-import argparse
-import os
 
-elements = { 'neut': 0, 'h': 1, 'he': 2, 'li': 3, 'be': 4, 'b': 5, 'c': 6, 'n': 7, 'o': 8, 'f': 9, 'ne': 10, 'na': 11, 'mg': 12, 'al': 13, 'si': 14, 'p': 15, 's': 16, 'cl': 17, 'ar': 18, 'k': 19,    'ca': 20, 'sc': 21, 'ti': 22, 'v': 23, 'cr': 24, 'mn': 25, 'fe': 26, 'co': 27, 'ni': 28, 'cu': 29, 'zn': 30, 'ga': 31, 'ge': 32, 'as': 33, 'se': 34, 'br': 35, 'kr': 36, 'rb': 37, 'sr': 38, 'y': 39,  'zr': 40, 'nb': 41, 'mo': 42, 'tc': 43, 'ru': 44, 'rh': 45, 'pd': 46, 'ag': 47, 'cd': 48}
-inv_elements = dict([(v,k) for k, v in elements.items()])
-
-def store_data_for_kromer_plot(mdl, lines_fname = "lines.hdf5", model_fname = "model.hdf5"):
+logger = logging.getLogger(__name__)
+
+elements = {'neut': 0, 'h': 1, 'he': 2, 'li': 3, 'be': 4, 'b': 5, 'c': 6, 'n':
```
Comment (wkerzendorf) [inline tardis_kromer_plot.py]: this list exists within TARDIS.

Comment (unoebauer) [inline tardis_kromer_plot.py]: Good to know - where exactly?

Comment (wkerzendorf) [inline tardis_kromer_plot.py]: https://github.com/tardis-sn/tardis/blob/master/tardis/data/atomic_symbols.dat

**Comment by unoebauer** [comment]
For now, I'll leave the hard-wired elements list - maybe in the futures it'll be replaced. Merging

**Pull request merged**